### DISCUSSION TITLE
How to start QEMU in Local PC or Laptop

### DISCUSSION THREAD

**Discussion by Deepak-Beniya** [OE4T/tegra-demo-distro#291] [Q&A] [answered]
I have selected machine as "qemux86-64" and compiled the source and the images are generated in build/tmp/deploy/images/qemux86-64

Now how will I start the the qemu in my local pc . Not in any jetson board .
I wanted to start the qemu in local pc or laptop.

what is the difference between these two machines "qemux86-64"  and  "qemuarm64" ?

How can I do customize the rootfs , I wanted to reduce size by removing cuda, and other library.?

Thanks and Regards 
Deepak Kumar Beniya

**Comment by madisongh** [comment, marked as answer]
`qemux86-64` emulates a system with an Intel/AMD 64-bit x86 CPU.  `qemuarm64` emulates a system with a 64-bit ARM architecture CPU.

The [Yocto Project documentation on running QEMU](https://docs.yoctoproject.org/singleindex.html#using-the-quick-emulator-qemu) talks about using QEMU, and the `runqemu` command.

As I mentioned in my comment on #169, using `qemuarm64` is probably a better choice for emulation, since the Jetsons use 64-bit ARM CPUs.  None of the NVIDIA-specific packages for the Jetsons (CUDA, etc.) will actually work in emulation.

Customizing images is discussed in [this section of the Yocto Project documentation](https://docs.yoctoproject.org/singleindex.html#customizing-images).

**Reply by Deepak-Beniya** [reply]
Thank you
